local-cluster/tests/local_cluster.rs

```diff
@@ -200,7 +200,7 @@ fn run_network_partition(partitions: &[usize]) {
         ..ClusterConfig::default()
     };
     let now = timestamp();
-    let partition_start = now + 30_000;
+    let partition_start = now + 60_000;
```

Comment:
Is the goal here to start the partition after all the validators are already in the leader schedule? If so, there's some logic in `cluster_tests::sleep_n_epochs` that can more precisely calculate how long to wait based on epoch len, ticks per slot, etc.

Reply:
https://github.com/solana-labs/solana/issues/7086